src/components/TreeReactFlow/index.tsx

```diff
@@ -259,6 +264,105 @@ const nodeTypes = {
       {handle("source", Position.Bottom)}
     </>
   ),
+  "primer-typedef-name": ({
```

Comment:
This has all become pretty repetitive. We should find a way to factor out the common parts. But for now, while we haven't decided how to style everything, this actually gives us a lot of flexibility.